astroquery/esa/euclid/core.py

```diff
@@ -0,0 +1,1205 @@
+        except Exception as exx:
+            log.error(f'Query failed: {query}, {str(exx)}')
+
+    def launch_job_async(self, query, *, name=None, dump_to_file=False, output_file=None, output_format="csv",
+                         verbose=False, background=False, upload_resource=None, upload_table_name=None, autorun=True):
+        """
+        Description
+        -----------
+        Launches an asynchronous job
+
+        Parameters
+        ----------
+        query : str, mandatory
+            query to be executed
+        name : str, optional, default None
+            custom name defined by the user for the job that is going to be created
+        dump_to_file : bool, optional, default 'False'
+            if True, the results are saved in a file instead of using memory
+        output_file : str, optional, default None
+            file name where the results are saved if dump_to_file is True.
+            if this parameter is not provided, the jobid is used instead
+        output_format : str, optional, default 'csv'
+            format of the results for the output file
+        verbose : bool, optional, default 'False'
+            flag to display information about the process
+        background : bool, optional, default 'False'
+            when the job is executed in asynchronous mode, this flag specifies whether
+            the execution will wait until results are available
+        upload_resource: str, optional, default None
+            resource to be uploaded to UPLOAD_SCHEMA
+        upload_table_name: str, required if uploadResource is provided, default None
+            resource temporary table name associated to the uploaded resource
+        autorun : boolean, optional, default True
+            if 'True', sets 'phase' parameter to 'RUN',
+            so the framework can start the job.
+
+        Returns
+        -------
+        A Job object
+        """
+        try:
+            return super().launch_job_async(query=query,
+                                            name=name,
+                                            output_file=output_file,
+                                            output_format=output_format,
+                                            verbose=verbose,
+                                            dump_to_file=dump_to_file,
+                                            background=background,
+                                            upload_resource=upload_resource,
+                                            upload_table_name=upload_table_name,
+                                            autorun=autorun,
+                                            format_with_results_compressed=('votable_gzip',))
+
+        except HTTPError as err:
+            log.error(f'Query failed: {query}: HTTP error: {err}')
+        except Exception as exx:
+            log.error(f'Query failed: {query}, {str(exx)}')
+
+    def query_object(self, coordinate, radius=None, width=None, height=None,
```

Comment:
make the parameters mandatory keyword params especially given their usage of either radius or width/height needed to be provided

```suggestion
    def query_object(self, coordinate, *, radius=None, width=None, height=None,
```

Reply:
I went ahead with this one, and then with the merging of the PR, so it's easier to shake out any remaining issues in potential follow-ups.